apps/sft/main.py

```diff
@@ -78,10 +77,13 @@ def __init__(self, config: DictConfig):
 
         self.current_step = 0
         self.num_training_steps = job_config.training.steps
+        self.metric_logger = None  # TODO: fix this
         self.gradient_accumulation_steps = 1  # Example value, adjust as needed
         self._rank = current_rank().rank
         self._size = math.prod(current_size().values())
+
         self._init_dist()
+
         super().__init__(job_config)
 
     def _init_dist(self):
```

Comment:
if we remove the _init_dist altogether would this still work? I added [this line](https://github.com/meta-pytorch/torchforge/blob/main/src/forge/controller/provisioner.py#L321) in `get_proc_mesh` later, so this should not be needed anymore. Could you please try it out?